dev/docker/3-ursulas.yml

```diff
@@ -0,0 +1,61 @@
+version: '3'
```

Comment:
So - more specifically, this script can possibly use a non-dev / standard requirements version of the codebase.

Reply:
yes exactly, or like we were talking about using postgres and we could have a version that pre-installs postgres and uses that.